Bingo_CSD_412/Models/Category.cs

```diff
@@ -10,12 +10,62 @@ public class Category
         //static category TODO: have to implement in Database 
         public static readonly Category CSD_412_Category = new Category();
         public String Name { get; set; }
-        public String[] Words { get; set; }
+        public HashSet<string> Words { get; set; }
+        public int Id { get; }
 
-        static Category()
+        static Category() //Kept this for testing purposes
         {
             CSD_412_Category.Name = "CSD_412";
-            CSD_412_Category.Words = new[] { "HTTP" , "URL", "Data Tier", "N Tier", "Application Settings" };
+            CSD_412_Category.Words.Add("HTTP");
+            CSD_412_Category.Words.Add("URL");
+            CSD_412_Category.Words.Add("Date Tier");
+            CSD_412_Category.Words.Add("N Tier");
+            CSD_412_Category.Words.Add("Application Settings");
+        }
+
+        /*public Category()  <-- Real constructor
+        {
+            Id = GenerateId();
+            Name = GetName();
+            Words = new HashSet<string>();
+        }*/
+
+        private string GetName()
```

Comment:
This is already implemented as a part of Name property - {get;set;}

Reply:
This is poor communication on my end, I meant to create a method that would retrieve user input and set the value of Category.Name = user input 

This method is poorly named but is intended to be used for initializing the Name property from the user.

Reply:
So it's more like CreateCategory method, that takes categoryName as input, and produces Category object as an output(and probably puts this new category into a database).
I think it should be a part of CategoryManager then.

Reply:
As long as Category can get the name from CategoryManager then that works!